### PR TITLE
4065 filter tags for user

### PR DESCRIPTION
**Pull request by wschaeferB**
Requires PR: #3984

### CODE REVIEW

File: Core/src/org/sleuthkit/autopsy/datamodel/ContentTagNode.java [hunk 1]
```diff
@@ -123,13 +128,14 @@ protected Sheet createSheet() {
         List<Action> actions = new ArrayList<>();
         actions.addAll(Arrays.asList(super.getActions(context)));
 
-        AbstractFile file = getLookup().lookup(AbstractFile.class);
+        AbstractFile file = getLookup().lookup(AbstractFile.class
+        );
```
Comment (dannysmyda) [inline Core/src/org/sleuthkit/autopsy/datamodel/ContentTagNode.java]: One line? Though I'm being really nitpicky.

Comment (wschaeferB) [reply Core/src/org/sleuthkit/autopsy/datamodel/ContentTagNode.java]: Oops thank you for catching this I will get it fixed. I must of auto formatted with a misplaced bracket or paren at some point, it tends to screw up stuff like this and the auto format doesn't correct it when done again after the bracket/paren is restored.

**Comment by dannysmyda** [comment]
- Ref Codacy/PR Quality Review Details

**Comment by wschaeferB** [comment]
Screwed up the merge somehow and wound up making this have 44 files, going to cherry-pick the changes over to a new PR

**Pull request closed without merging**